### PR TITLE
release plan

### PR DESCRIPTION
**Pull request by j-zimnowoda**

### CODE REVIEW

File: product/roadmap.md [hunk 1]
```diff
@@ -50,10 +53,50 @@ We are dedicated to maintain support for three different Kubernetes versions wit
...
 - Remove platform app forms in favor of generated values that can be customized in an editor
+
+## Removed features by release
+
+### v3.0
+
+#### Drone
+
+Otomi will stop delivering Drone as a platform app.
+The Tekton app is introduced in Otomi v2.0 as a replacement. There is no migration procedure, Drone is used primarely as an Otomi deployment pipeline. However if you used it as for custom pipeline then you may need to migrate them Tekton.
```
Comment (ElderMatt) [inline product/roadmap.md:71]: `it as for custom pipeline` -> `it for custom pipelines`

File: product/roadmap.md [hunk 2]
```diff
@@ -50,10 +53,50 @@ We are dedicated to maintain support for three different Kubernetes versions wit
...
+
+:::note
+This app will not be deleted if it is already deployed.
+The external-secrets app is still going to be part of Otomi
+:::
+
+#### OPA Gatekeeper
+
+Otomi will stop delivering the OPA Gatekeeper as a platform app.
+The Kyverno app is introduced in Otomi v2.7 as a replacement. There is no migration procedure, as the basic policies are already covered. However if you used your custom policies then you may need to migrated them on your own.
```
Comment (ElderMatt) [inline product/roadmap.md:91]: `However if you used your custom policies then you may need to migrated them on your own.` -> `However if you used it for custom policies then you may need to migrate them on your own.`

**Review by ElderMatt — CHANGES_REQUESTED** [review]
To make the sentences flow better.

File: product/roadmap.md (hunk 1 above)
Comment (j-zimnowoda) [reply product/roadmap.md:71]: ok

File: product/roadmap.md (hunk 2 above)
Comment (j-zimnowoda) [reply product/roadmap.md:91]: ok

**Review by srodenhuis — APPROVED** [review]

**Pull request merged**